menu.cpp

```diff
@@ -0,0 +1,108 @@
+#include "menu.h"
+
+bool end = false;
+
+void ShowMenu()
+{
+	puts("-------------------------------------------------------------");
+	puts("1 - Initialize string list");
+	puts("2 - Insert value at the end of the list");
+	puts("3 - Remove all occurrences of str in the list");
+	puts("4 - Return the number of items in the list");
+	puts("5 - Return the index position of str in the list");
+	puts("6 - Remove duplicate entries from the list");
+	puts("7 - Replace every occurrence of before in the list with after");
+	puts("8 - Sort the list of strings in ascending order");
+	puts("9 - Print the list of strings to the console");
+	puts("0 - Clear list and exit the program");
+	puts("-------------------------------------------------------------");
+}
+
+void InputChoice()
+{
+	int choice;
+	printf("Enter your choice: ");
+	scanf_s("%d", &choice);
+	while (choice < 0 || choice > 9)
+	{
+		printf("Wrong choice! Enter again: ");
+		scanf_s("%d", &choice);
+	}
+
+	SelectFunction(choice);
+}
+
+void SelectFunction(const int choice)
+{
+	if (choice == 1)
```

Comment:
This looks like a 'switch' statement not 'if else'. Can we rewrite this to 'switch'?

Reply:
Yes, we can. But for me if 'switch' does not fit on one page, we need to scroll up to see what variable are using to compare. Also we don't need to forget to place 'break' almost everywhere, so that 'switch' becomes huge.